src/imitation/algorithms/preference_comparisons.py

```diff
@@ -176,7 +186,7 @@ def __init__(
         self.algorithm.set_env(self.venv)
         policy_callable = rollout._policy_to_callable(
             self.algorithm,
-            self.venv,
+            self.algorithm.get_env(),
```

Comment:
we can't use `self.venv` here or else `_policy_to_callable` will complain about observation spaces not matching up between `self.algorithm`'s transposed observation space vs. `self.venv`'s untransposed observation space

Reply:
Thanks for flagging. Perhaps turn this into a one-line comment in the code as others may wonder about this in the future?

This is a bit awkward but I don't think we can do much to improve this without modifying SB3.